src/components/ResourcesCards.js

```diff
@@ -8,6 +10,9 @@ const NewMember = "https://shpeuf.s3.amazonaws.com/public/resources/newmember.jp
 const WhatsApp = "https://shpeuf.s3.amazonaws.com/public/resources/whatsapp.png";
 const InfoSlides = "https://shpeuf.s3.amazonaws.com/public/resources/info.jpg";
 const GBM = "https://shpeuf.s3.amazonaws.com/public/misc/gbm.JPG";
+const resourceImage = "https://shpeuf.s3.amazonaws.com/public/resources/resourceImage.png"
```

Comment:
can the resourceImage be a part of the schema for a resource? That way when we run FETCH_RESOURCE_QUERY, we can read the S3 object url from there. And then in the ResourcesMenu, this default link can be hardcoded for now.